lib/api/docs.rb

```diff
@@ -1,3 +1,4 @@
 module Api
-  Docs = ::OpenApi::Docs.new(Dir.glob(Rails.root.join("public/doc/swagger*.yaml")))
+  Docs = ::OpenApi::Docs.new(Dir.glob(Rails.root.join("public", "doc", "openapi*.json")) +
+                             Dir.glob(Rails.root.join("public", "doc", "swagger*.yaml")))
```

Comment:
Can we just drop the yamls?

Reply:
Or put another way, can we just drop v2 altogether, since we are going to cut a 1.0 anyway

Reply:
We can drop the yamls & v2 Doc library since we don't expose /api/v0.0/openapi.json anyways,  want me to do in this PR ?

Reply:
yup 👍